src/video/opengl.cpp

```diff
@@ -1174,7 +1174,7 @@ uint8 *OpenGLBackend::GetAnimBuffer()
 	if (this->anim_pbo == 0) return nullptr;
 
 #ifndef NO_GL_BUFFER_SYNC
-	if (this->sync_anim_mapping != nullptr) _glClientWaitSync(this->sync_anim_mapping, GL_SYNC_FLUSH_COMMANDS_BIT, 10000000);
+	if (this->sync_anim_mapping != nullptr) _glClientWaitSync(this->sync_anim_mapping, GL_SYNC_FLUSH_COMMANDS_BIT, 100000000); // 100ms timeout.
```

Comment:
Maybe `100 * 1000 * 1000` to make us count less zeros? Feel free to say no :)

Reply:
Hey, I did add a comment 😀